test/runtests.jl

```diff
@@ -3270,6 +3274,44 @@ else  # run HiGHS tests
             @test s.space_heating_load.loads_kw[end-24+1:end] == s.space_heating_load.loads_kw[1:24]
             @test s.cooling_load.loads_kw_thermal[end-24+1:end] == s.cooling_load.loads_kw_thermal[1:24]
         end
-        
+
+        @testset "After-tax savings and capital cost metric for alternative payback calculation" begin
+            """
+            Check alignment between REopt simple_payback_years and a simple X/Y payback metric with
+            after-tax savings and a capital cost metric with non-discounted incentives to get simple X/Y payback 
+            The REopt simple_payback_years output metric is after-tax, with no discounting, but it uses escalated and 
+            inflated cashflows and it includes out-year, non-discounted battery replacement cost which is only included 
```

Comment:
noting here too that Generator can also be replaced

Reply:
Yes, but punting on that addition for now, as noted in the other comment. I'll add an issue on GitHub for a future improvement.